pkg/tasks/alertmanager.go

```diff
@@ -49,19 +49,21 @@ func (t *AlertmanagerTask) Run(ctx context.Context) error {
 }
 
 func (t *AlertmanagerTask) create(ctx context.Context) error {
-	r, err := t.factory.AlertmanagerRoute()
-	if err != nil {
-		return errors.Wrap(err, "initializing Alertmanager Route failed")
-	}
+	if _, err := t.client.GetClusterOperator(ctx, "ingress"); err != nil {
```

Comment:
hmm why testing for `err != ni` here? I'd expect that we check for `apierrors.IsNotFound(err)` instead. Maybe a small wrapper function (e.g. `IngressOperatorPresent()`  would make it more readable.

Reply:
:+1: good point yes

Reply:
Added.